plugins/heindekort/info.json

```diff
@@ -0,0 +1,10 @@
+{
+  "name": "Hein de Kort",
+  "color": "d8d8d8",
```

Comment:
Instead of using yet another gray, maybe you can use the yellow used on its website or the blue you have in the cover